calendar_reboot.js

```diff
@@ -243,15 +246,13 @@ $(document).ready(function() {
         if (confirmation === true) {
             
             var currentCalendarId = loadFromLocalStorage(current_active_calendar);
-            console.log(currentCalendarId);
-            delete calendarUniqueId[currentCalendarId];
-            storeInLocalStorage(masterKey, calendarUniqueId);
-            
             removeFromCalendarDropdown(currentCalendarId);
-            //remove the calendarstate from localStorage
-            removeFromLocalStorage(currentCalendarId);
             
-            removeFromLocalStorage(current_active_calendar);
+            
+            //below does not work
```

Comment:
Oh. Yes, go ahead. The calendarState is the essence of the calendar. The calendar object is just a wrapper that lets you do stuff with it. You can make and destroy it as needed. Two calendar objects made from the same state *should* be interchangeable.